lib/playground.dart

```diff
@@ -766,7 +767,11 @@ class Playground extends EditorUi implements GistContainer, GistController {
     queryParams.gistId = gistId;
   }
 
+
   void _showGist(String gistId) {
+    if (_gistIdInProgress != null && _gistIdInProgress == gistId) {
```

Comment:
Seems like `gistId` will never be `null` anyway, so can simplify to this:

```suggestion
    if (_gistIdInProgress == gistId) {
```